src/components/StatTile.vue

```diff
@@ -10,177 +10,194 @@
       '-sq-footage': isSquareFootage,
     }"
   >
-    <SparkLine
```

Comment:
Check hide whitespace under the diff options (gear icon) and this will be a lot cleaner - I mostly wrapped the text so I could use a flexbox layout rather than the more crude `float: right` on the stat line